test/sanity/run-test-v2.sh

```diff
@@ -22,6 +22,10 @@ echo image
 function cleanup {
   set +e
 
+  echo 'Uninstalling CRD'
+  kubectl delete crd azvolumes.disk.csi.azure.com
```

Comment:
Can we reuse the cleanup template for deleting the crds and cris here for test as well? If not, we should make these _kubectl delete_ changes for _test/integration/run-test-v2.sh_ as well.